widget/widget.go

```diff
@@ -163,6 +169,21 @@ func (w *BaseWidget) super() fyne.Widget {
 	return impl
 }
 
+// theme returns a cached theme instance for this widget (or its extending widget).
+func (w *BaseWidget) theme() fyne.Theme {
+	w.propertyLock.RLock()
+	cached := w.themeCache
+	w.propertyLock.RUnlock()
+	if w.themeCache == nil {
```

Comment:
```suggestion
	if cached == nil {
```

Reply:
Oh, missed that before sorry - it's in now